source/server/http/admin.cc

```diff
@@ -275,9 +276,9 @@ Http::Code AdminImpl::handlerClusters(const std::string&, Http::HeaderMap&,
   return Http::Code::OK;
 }
 
-Http::Code AdminImpl::handlerCpuProfiler(const std::string& url, Http::HeaderMap&,
+Http::Code AdminImpl::handlerCpuProfiler(absl::string_view url, Http::HeaderMap&,
                                          Buffer::Instance& response) {
-  Http::Utility::QueryParams query_params = Http::Utility::parseQueryString(url);
+  Http::Utility::QueryParams query_params = Http::Utility::parseQueryString(std::string(url));
```

Comment:
You can avoid this conversion/allocation if you push one level deeper into this function. I think it would be straightforward without affecting other functions, at least at first glance. But up to you if you want to do that or not.

Reply:
Yeah I didn't need that construction.  Good catch.